test/functional/legacy/076_completefunc_spec.lua

```diff
@@ -0,0 +1,66 @@
+-- Tests for completefunc/omnifunc.
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, expect = helpers.clear, helpers.expect
+
+describe('completefunc', function()
+  setup(clear)
+
+  it('is working', function()
+    insert([=[
+      +++
+      one
+      two
+      three]=])
+
+    -- Test that nothing happens if the 'completefunc' opens.
+    -- A new window (no completion, no crash).
+    source([=[
+      function! DummyCompleteOne(findstart, base)
```

Comment:
There's something strange going on with tabs/spaces in all function bodies. The tabs aren't required here, you can replace them with spaces.